app/bundles/CoreBundle/Form/RequestTrait.php

```diff
@@ -51,9 +50,7 @@ protected function prepareParametersFromRequest(FormInterface $form, array &$par
             }
 
             $type = $child->getConfig()->getType();
-            if ($type instanceof ResolvedFormTypeInterface) {
-                $type = $type->getInnerType();
-            }
+            $type = $type->getInnerType();
```

Comment:
This change seems wrong to me. Either this should be 
`$type = $child->getConfig()->getType()->getInnerType();`
Or be as it was before.